src/main/java/io/github/moulberry/notenoughupdates/overlays/SlayerOverlay.java

```diff
@@ -117,6 +119,22 @@ public void update() {
 			isSlayerNine = true;
 		} else if (!slayerXp.equals("0")) {
 			slayerEXP = slayerXp.replace(",", "");
+
+			differenceFromLastXP = slayerIntXP-Integer.parseInt(slayerEXP);
+			if(differenceFromLastXP != 0){
```

Comment:
Please reformat your code correctly. What IDE do you use? When Intellij, just press control +alt + L. If you use VSC or something else, just manually add spaces

Reply:
![grafik](https://user-images.githubusercontent.com/24389977/201478788-9c4fe6a6-739b-4c28-a027-20348ef9315d.png)

Reply:
> Please reformat your code correctly. What IDE do you use? When Intellij, just press control +alt + L. If you use VSC or something else, just manually add spaces

I do use Intellij and I downloaded it a few days ago so I am not that familiar with it yet, thank you for showing me the reformat keybind.

Reply:
Happy to help :)